Server/src/server.cpp

```diff
@@ -1660,30 +1729,33 @@ int Server::addStartup() {
  * @param command The authentication command received from the client.
  * @return 0 on successful authentication, -1 on error or authentication failure.
  */
-int Server::handleAuth(char *command) {
+int Server::handleAuth(char* command) {
     shiftStrLeft(command, 6);
     std::string username, password;
     int space_counter = 0;
     bool second_word = false;
 
-    for(int i = 0, len = (int) strlen(command); i < len; i++) {
-        if(command[i] == ' ') {
+    for (int i = 0, len = (int)strlen(command); i < len; i++) {
+        if (command[i] == ' ') {
             space_counter++;
             second_word = true;
         }
 
-        if(second_word)
+        if (second_word)
             password += command[i];
         else
             username += command[i];
     }
 
-    if(space_counter != 1) return -1;
+    if (space_counter != 1) return -1;
 
     int res = auth(username, password);
-    if(res == -1) return -1;
+    if (res == -1) return -1;
 
-    if(handleSend("valid") != 0) return -1;
+    userMap[LastSock] = username;
+    std::cout << "Accepted new client. Username: " << username << std::endl;
```

Comment:
Could you log this as well? Makes it easier to see where something gooes wrong if it does